### DISCUSSION TITLE
Feature Request:  Normalized PageRank

### DISCUSSION THREAD

**Discussion by networks1** [networkit/networkit#685] [Ideas]
I would like to suggest that normalization of page rank be offered, as described [in this paper](https://domino.mpi-inf.mpg.de/intranet/ag5/ag5publ.nsf/0/31deb7636690f704c125729c003181ef/$file/www2007.pdf).

**Comment by angriman** [comment]
Thank you for the suggestion, @Relux-the-Relux implemented it in #720.

**Reply by BradKML** [reply]
Making notes on alternative Centrality issues in https://github.com/networkit/networkit/discussions/824
